src/uu/uniq/src/uniq.rs

```diff
@@ -390,12 +411,24 @@ pub fn uu_app() -> Command {
                 .help("end lines with 0 byte, not newline")
                 .action(ArgAction::SetTrue),
         )
+        .group(
+            ArgGroup::new("obsolete_skip_field")
+            .multiple(false)
```

Comment:
Generally, coreutils arguments can be used multiple times (with overriding behaviour), but since our behaviour here would be different from GNU uniq, what you got now is probably better. However, I do think we need to add comment about that here.